### PR TITLE
Update setup: Add kivy dir to packages

### PR DESCRIPTION
**Pull request by shekhar-cis**
Update setup: Add kivy dir to packages

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -138,7 +145,7 @@ def run(self):
         package_dir={'pybitmessage': 'src'},
         packages=packages,
         package_data={'': [
-            'bitmessageqt/*.ui', 'bitmsghash/*.cl', 'sslkeys/*.pem',
+            'bitmessageqt/*.ui', 'bitmsghash/*.cl', 'sslkeys/*.pem', 'bitmessagekivy/*.kv',
```
Comment (PeterSurda) [inline setup.py:148]:
```
'bitmessagekivy/main.kv', 'bitmessagekivy/kv/*.kv',
```

File: setup.py [hunk 2]
```diff
@@ -72,11 +72,18 @@ def run(self):
...
+        'pybitmessage.mock'
     ]
 
     if sys.version_info[0] == 3:
-        packages.extend(['pybitmessage.bitmessagekivy', 'pybitmessage.bitmessagekivy.baseclass'])
+        packages.extend(
+            [
+                'pybitmessage.bitmessagekivy',
+                'pybitmessage.bitmessagekivy.baseclass',
+                'pybitmessage.bitmessagekivy.kv'
```
Comment (PeterSurda) [inline setup.py:84]: this is not a python module

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (shekhar-cis) [reply setup.py]: Done

File: setup.py (hunk 2 above)
Comment (shekhar-cis) [reply setup.py]: Done

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**